docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,10 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Changed the error handling so it returns the error message and the related
```

Comment:
We currently don't consider error messages to be stable, so I think this could be moved to the changes section instead of the breaking changes section.